src/Nether.Analytics/Results/IResultsStore.cs

```diff
@@ -0,0 +1,18 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Nether.Analytics.Results
+{
+    /// <summary>
+    /// Defines the interface for communicating with the results store. 
+    /// </summary>
+    public interface IResultsStore
+    {
+        /// <summary>
+        /// Gets the result query for a specific pipeline.
+        /// </summary>
+        /// <param name="pipeline"></param>
```

Comment:
Since we are moving away from "pipelines" in the folder structure, make sure to not use pipeline as the name for jobs.